modules/serverless-installing-service-mesh-member-roll.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+[id="installing-service-mesh-member-roll_{context}"]
+= Installing a ServiceMeshMemberRoll
+
+For applications to make use of the deployed control plane and ingress, their namespaces need to be part of a member roll. 
+
+.ServiceMeshMemberRoll Custom Resource Example
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+ name: default
+ Namespace: istio-system
+spec:
+ members:
+ - knative-serving
+ - mynamespace
+----
+
+
+A multi-tenant control plane installation only affects namespaces configured as part of the Service Mesh. You must specify the namespaces associated with the Service Mesh in a `ServiceMeshMemberRoll` resource located in the same namespace as the `ServiceMeshControlPlane` resource and name it `default`.
+
+
+.Prerequisites
+* An installed, verified Service Mesh Operator.
+* A custom resource file that defines the parameters of your Red Hat OpenShift Service Mesh control plane.
+
+
+.Procedure
+. Create a YAML file that replicates the ServiceMeshMemberRoll Custom Resource sample.
```

Comment:
show all the examples here with the actual procedure steps to give context, random examples with no steps don't have enough context

Reply:
@abrennan89  I don't understand what you mean by this. Would you like me to include the YAML file in the procedure? Looking at OpenShift docs examples, it seems they have the YAML examples outside of the procedures

Reply:
If you're including the example outside the procedure, I'd make sure it's explained with some context about what individual fields are required for basically.
E.g. in the service mesh docs here https://docs.openshift.com/container-platform/4.1/service_mesh/service_mesh_install/installing-ossm.html there are tables which explain what the fields do.
I'm not sure we need to explain every field in a yaml example because some of it is probably assumed knowledge or elsewhere in docs, like at the link above that you could point to, but I think it'd be better to try to call out or draw attention to what's actually different for serverless.

LMK if that makes sense. @markusthoemmes any thoughts? I think you mentioned something similar in another comment re calling out the important fields in sample yaml?